### ISSUE TITLE
Merge deploy_data_specific_image script with the deploy-data script

### ISSUE THREAD

**Issue by ChaamC** [bird-house/birdhouse-deploy#169]
The [deploy_data_specific_image script](https://github.com/bird-house/pavics-jupyter-base/blob/master/scheduler-jobs/deploy_data_specific_image) found on the pavics-jupyter-base repo was initially implemented as a separate script from the [deploy-data script](https://github.com/bird-house/birdhouse-deploy/blob/57a320c3583f804838e8c18aeb064266527a0bc9/birdhouse/deployment/deploy-data).

It made more sense at first since the specific image script is required to run directly on the specific images, using .yaml config files stored on the images, as opposed to the deploy-data script from birdhouse, which runs on a generic image that has git and docker installed, using .env config files found directly on the birdhouse repo ( [example of .env file](https://github.com/bird-house/birdhouse-deploy/blob/57a320c3583f804838e8c18aeb064266527a0bc9/birdhouse/components/scheduler/deploy_raven_testdata_to_thredds.env) )

Since both scripts are pretty similar in the kind of task they do, `deploy_data_specific_image` could be merged into the more generic `deploy-data` script.

To do (non-exhaustive, taken from this [comment](https://github.com/bird-house/birdhouse-deploy/pull/150#discussion_r617057555)) :
- being able to use yq as docker run (deploy-data) or as installed (deploy_data_specific_image), will have to add a new switch for that.
- download the folders with svn (deploy_data_specific_image) instead of git pull (deploy-data), will have to add a new switch for that as well.
- Determine if we should add a switch between the svn and git pull/rsync method, or if we should just use one of those methods. After implementing the `deploy_data_specific_image` script, some limitations were observed. For example, it didn't seem easy to use * tokens in the svn command, to download only the notebooks for example with the format "*.ipynb". Also, the current `deploy_data_specific_image` script is not made to download single files for now, although it is possible and not hard to add as a feature. The script only supports downloading a full folder's content and sending it to the destination folder.

In conclusion, it would be nice to improve this part, but it is not required for now, and not a priority.
